jade.jade

```diff
@@ -0,0 +1,78 @@
+# == Class: automysqlbackup
```

Comment:
This looks like a puppet file to me, and given that it's the same as the `puppet.pp` is it possible this was a mistaken copy paste?

Reply:
Ah yes, it appears that it is. I will fix it.

Reply:
Ok, it has been removed now.